include/z64save.h

```diff
@@ -1108,10 +1108,10 @@ typedef enum {
 #define WEEKEVENTREG_60_01 PACK_WEEKEVENTREG_FLAG(60, 0x01)
 #define WEEKEVENTREG_60_02 PACK_WEEKEVENTREG_FLAG(60, 0x02)
 #define WEEKEVENTREG_60_04 PACK_WEEKEVENTREG_FLAG(60, 0x04)
-#define WEEKEVENTREG_60_08 PACK_WEEKEVENTREG_FLAG(60, 0x08)
-#define WEEKEVENTREG_60_10 PACK_WEEKEVENTREG_FLAG(60, 0x10)
+#define WEEKEVENTREG_ATTENDED_MAYOR_MEETING PACK_WEEKEVENTREG_FLAG(60, 0x08)
+#define WEEKEVENTREG_RECEIVED_MAYOR_REWARD PACK_WEEKEVENTREG_FLAG(60, 0x10)
 #define WEEKEVENTREG_60_20 PACK_WEEKEVENTREG_FLAG(60, 0x20)
-#define WEEKEVENTREG_60_40 PACK_WEEKEVENTREG_FLAG(60, 0x40)
+#define WEEKEVENTREG_SPOKE_WITH_MAYOR_ON_NIGHT_3 PACK_WEEKEVENTREG_FLAG(60, 0x40)
```

Comment:
```suggestion
#define WEEKEVENTREG_TALKED_MAYOR_NIGHT_3 PACK_WEEKEVENTREG_FLAG(60, 0x40)
```
We have standardized these kinds of week event regs.